components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -1,16 +1,12 @@
-phenotips.patientSharing.push.dialog.serverSelection.label.multiplePatients=Where to send all the patient data to?
-phenotips.patientSharing.push.dialog.serverSelection.hint.multiplePatients=Remote PhenoTips server where all the data will be sent.
-phenotips.patientSharing.push.dialog.serverSelection.label.singlePatient=Where to send this patient profile?
-phenotips.patientSharing.push.dialog.serverSelection.hint.singlePatient=Remote PhenoTips server where this patient's data will be sent.
-phenotips.patientSharing.push.dialog.authentication.label=Authentication on remote server
-phenotips.patientSharing.push.dialog.authentication.hint=Please choose the user name to be used to authenticate on the server and send the data; this user will be marked as the reporter on the remote server.
-phenotips.patientSharing.push.dialog.remoteGroup.label=Remote PhenoTips group
-phenotips.patientSharing.push.dialog.remoteGroup.hint=The PhenoTips group you would like to share this patient with on the remote server. If a group is selected, the group would be the owner and the given remote user will be a collaborator. If no group is selected, the remote user will be the only owner.
+phenotips.patientSharing.push.dialog.authentication.label=Log in to __serverID__
+phenotips.patientSharing.push.dialog.authentication.hint=Please log in to __serverID__ using your __serverID__ username and password.
+phenotips.patientSharing.push.dialog.remoteGroup.label=Would you like to add this patient to one of __serverID__ groups?
+phenotips.patientSharing.push.dialog.remoteGroup.hint=If a group is selected, the group will be marked as the owner of the patient record and you will be marked as a collaborator. If no group is selected you will be marked as the owner of the patient record..
 phenotips.patientSharing.push.dialog.preview.label=Patient data preview
 phenotips.patientSharing.push.dialog.preview.hint=Please review the data that will be sent to the server.
 phenotips.patientSharing.push.dialog.confirmation.label=I agree to push selected patient data to
-phenotips.patientSharing.push.dialog.title.multiplePatients=Pushing patient data to a remote PhenoTips instance\u2026
-phenotips.patientSharing.push.dialog.title.singlePatient=Pushing __patientID__ to a remote database\u2026
+phenotips.patientSharing.push.dialog.title.multiplePatients=Pushing patient data to __serverID__\u2026
+phenotips.patientSharing.push.dialog.title.singlePatient=Pushing __patientID__ to __serverID__\u2026
```

Comment:
Do these modified strings need to be removed from property files for other languages?